backend/WikiContrib/.gitignore

```diff
@@ -1,9 +1,5 @@
 static_cdn/
-query/migrations/__pycache__/
-result/migrations/__pycache__/
```

Comment:
Please do not remove them. It will result __pycahce__ directories to be committed.

Reply:
About this @rammanoj , below are the various __pycache__ directories that where initially on the .gitignore file:
query/migrations/__pycache__/
result/migrations/__pycache__/
query/__pycache__/
WikiContrib/__pycache__/
result/__pycache__/
query/management/__pycache__/

Before submitting this PR, i discovered i needed to also add "query/management/commands/__pycache__/" to the .gitignore as well. So instead of all these directories, I removed all and replaced them with just "__pycache__/" which still basically does the same thing